position/errors.gno

```diff
@@ -0,0 +1,24 @@
+package position
+
+import (
+	"errors"
+
+	"gno.land/p/demo/ufmt"
+)
+
+var (
+	errNoPermission  = errors.New("[GNOSWAP-POSITION-001] no permission")
+	errSlippage      = errors.New("[GNOSWAP-POSITION-002] slippage")
+	errWrapUnwrap    = errors.New("[GNOSWAP-POSITION-003] wrap/unwrap")
```

Comment:
On what occasion will this occur?

Reply:
same as https://github.com/gnoswap-labs/gnoswap/pull/355#discussion_r1811787519